### PR TITLE
feat(cli): add `aptu pr queue` subcommand

### PR DESCRIPTION
**Pull request by clouatre**
## Summary

Adds `aptu pr queue [--repo REPO] [--limit N]` -- a ranked advisory list of open PRs for a repository. No writes of any kind.

PRs are scored and sorted by reviewability: smaller and older PRs surface first. Draft PRs are excluded entirely. The command is a single API call (no N+1 per-PR fetches), so it is fast and rate-limit-safe.

## Scoring formula

```
score = 0.6 * (1 - normalized_size) + 0.4 * age_normalized
```

- `normalized_size = 1.0 - min(additions + deletions, max_size) / max_size` (max_size = max across all PRs, floor 500)
- `age_normalized = (days_since_creation / 365).clamp(0, 1)`
- Ties broken by PR number (ascending)

Documented in `--help` output.

## Changes

- `crates/aptu-cli/src/cli.rs` -- `PrCommand::Queue { repo, limit }` variant with Clap derive
- `crates/aptu-cli/src/commands/pr.rs` -- `compute_score()` pure fn + `run_queue()` handler with `all_pages()` pagination
- `crates/aptu-cli/src/commands/mod.rs` -- `PrCommand::Queue` match arm in `run_pr_command`
- `crates/aptu-cli/src/output/pr.rs` -- `QueuedPr` + `PrQueueResult` structs with `Renderable` (text table + JSON)
- `crates/aptu-cli/src/output/mod.rs` -- expose `pr` module as `pub mod`
- `crates/aptu-core/src/github/mod.rs` -- re-export `create_client` as `pub use`

## Test plan

- [x] 6 new unit tests: `compute_score` happy path, zero-lines edge case, brand-new PR, age cap at 1 year, sort stability on tied scores, limit=0 returns all
- [x] 80 tests pass total (`cargo test -p aptu-cli`)
- [x] Lint clean (`cargo clippy -p aptu-cli -- -D warnings`)
- [x] Security scan clean (aptu scan_security, 0 findings)

## Notes

CI status and conflict detection deferred to a follow-up PR (TODO comments in `commands/pr.rs`). This PR establishes the output types, scoring foundation, and test patterns.

### CODE REVIEW

File: crates/aptu-cli/src/commands/pr.rs [hunk 1]
```diff
@@ -292,11 +292,230 @@ pub async fn run_label(
...
+/// (then by number ASC for ties), and applies limit.
+///
+/// TODO: In follow-up PR, add CI status and conflict detection.
+/// TODO: In follow-up PR, add caching of results.
+#[instrument(skip_all, fields(repo, limit))]
+pub async fn run_queue(
+    _config: &aptu_core::AppConfig,
+    owner: &str,
+    repo: &str,
+    limit: u32,
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/commands/pr.rs:333]: Fixed. Added `MAX_QUEUE_PRS = 200` cap with a `warn` log. PRs beyond the cap are truncated before scoring. Also added `per_page(100)` to minimize round trips.

File: crates/aptu-cli/src/commands/pr.rs [hunk 2]
```diff
@@ -292,11 +292,224 @@ pub async fn run_label(
...
+///
+/// Fetches all open PRs, excludes drafts, computes scores, sorts by score DESC
+/// (then by number ASC for ties), and applies limit.
+///
+/// TODO: In follow-up PR, add CI status and conflict detection.
+/// TODO: In follow-up PR, add caching of results.
+#[instrument(skip_all, fields(repo, limit))]
+pub async fn run_queue(
+    _config: &aptu_core::AppConfig,
+    owner: &str,
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/commands/pr.rs:331]: Addressed by the `MAX_QUEUE_PRS = 200` cap added in the previous fix. The cap bounds both memory and API time; a full tokio timeout would add complexity without meaningfully improving the UX for this advisory command.

File: crates/aptu-cli/src/output/pr.rs [hunk 3]
```diff
@@ -13,6 +14,160 @@ use crate::commands::types::{
...
+    pub score: f64,
+    /// Is draft
+    pub draft: bool,
+}
+
+/// Result from `pr queue` command.
+#[derive(Debug, Serialize)]
+pub struct PrQueueResult {
+    /// Queued PRs sorted by score DESC
+    pub prs: Vec<QueuedPr>,
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/output/pr.rs:45]: Not applicable here. `age_days` is derived from `chrono::Duration::num_days()` which returns an integer, so the value is already a whole number -- `round()` and `floor()` produce the same result.

File: crates/aptu-cli/src/output/pr.rs [hunk 4]
```diff
@@ -13,6 +14,160 @@ use crate::commands::types::{
...
+
+        for (idx, pr) in self.prs.iter().enumerate() {
+            let rank = idx + 1;
+            let title = aptu_core::utils::truncate(&pr.title, 50);
+            let age_str = format_age(pr.age_days);
+            let changes = format!("+{}-{}", pr.additions, pr.deletions);
+            #[allow(clippy::cast_possible_truncation, clippy::cast_sign_loss)]
+            let score_int = (pr.score * 100.0).round() as u32;
+
+            writeln!(
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/output/pr.rs:100]: Out of scope for this PR. Adding a new crate dependency for terminal-width detection would be a YAGNI addition on a first-pass advisory command. Fixed-width truncation is consistent with how other aptu commands render tabular data (e.g. issue list). Can revisit if user feedback identifies it as a pain point.

File: crates/aptu-cli/src/commands/pr.rs [hunk 5]
```diff
@@ -292,11 +292,249 @@ pub async fn run_label(
...
+/// * `deletions` - Number of lines deleted
+/// * `age_days` - Age in days
+/// * `max_size` - Maximum size encountered (floor at 500)
+///
+/// # Returns
+///
+/// Score in [0.0, 1.0]
+#[allow(clippy::cast_precision_loss)]
+pub fn compute_score(additions: u64, deletions: u64, age_days: f64, max_size: u64) -> f64 {
+    let max = max_size.max(500);
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/commands/pr.rs]: Applied the suggestion. Added `const MIN_MAX_SIZE: u64 = 500` and replaced the inline literal.

File: crates/aptu-cli/src/output/pr.rs (hunk 4 above)
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/output/pr.rs:100]: Declining: this is consistent with existing tabular output in aptu (issues, models, repo list) which all use fixed-width columns. Adding a terminal-width dependency (tabled or termsize) is out of scope for this PR. The 120-char separator line is decorative and does not truncate any data; title truncation is already at 50 chars.

File: crates/aptu-cli/src/output/pr.rs [hunk 6]
```diff
@@ -13,6 +14,148 @@ use crate::commands::types::{
...
+        writeln!(w, "Ranked by reviewability score (60% size, 40% age)\n")?;
+
+        writeln!(w, "| Rank | Title | Author | Age | Changes | Score |")?;
+        writeln!(w, "|------|-------|--------|-----|---------|-------|")?;
+
+        for (idx, pr) in self.prs.iter().enumerate() {
+            let rank = idx + 1;
+            let title = aptu_core::utils::truncate(&pr.title, 50);
+            let age_str = format_age(pr.age_days);
+            let changes = format!("+{}-{}", pr.additions, pr.deletions);
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/output/pr.rs:137]: Fixed. Extracted to `const QUEUE_TITLE_MAX_CHARS: usize = 50` in `output/pr.rs`. Both `truncate` call sites now reference the constant.

File: crates/aptu-cli/src/commands/pr.rs [hunk 7]
```diff
@@ -292,11 +292,290 @@ pub async fn run_label(
...
+///
+/// # Returns
+///
+/// Score in [0.0, 1.0]
+#[allow(clippy::cast_precision_loss)]
+pub fn compute_score(additions: u64, deletions: u64, age_days: f64, max_size: u64) -> f64 {
+    const MIN_MAX_SIZE: u64 = 500;
+    let max = max_size.max(MIN_MAX_SIZE);
+    let age = age_days.max(0.0);
+    let total_changes = additions.saturating_add(deletions);
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/commands/pr.rs:315]: Applied the suggestion verbatim. The chain is now `max_size.max(MIN_MAX_SIZE).max(1)`, making the non-zero invariant explicit regardless of how `max_size` is derived in future callers.

File: crates/aptu-cli/src/commands/pr.rs [hunk 8]
```diff
@@ -292,11 +292,290 @@ pub async fn run_label(
...
+///
+/// # Returns
+///
+/// Score in [0.0, 1.0]
+#[allow(clippy::cast_precision_loss)]
+pub fn compute_score(additions: u64, deletions: u64, age_days: f64, max_size: u64) -> f64 {
+    const MIN_MAX_SIZE: u64 = 500;
+    let max = max_size.max(MIN_MAX_SIZE).max(1);
+    let age = age_days.max(0.0);
+    let total_changes = additions.saturating_add(deletions);
```
[the remark this answers is not in the document]
Comment (clouatre) [reply crates/aptu-cli/src/commands/pr.rs:315]: Declining. `MIN_MAX_SIZE` is already a named constant (not a bare magic number), which satisfies the maintainability concern. Making this value user-configurable would add config schema complexity and a user-facing knob for a scoring heuristic that has no established baseline -- YAGNI for an MVP advisory command. Can revisit if user feedback suggests a need to tune the threshold.
